lib/kernel/spir_layer.ll

```diff
@@ -0,0 +1,2668 @@
+; ModuleID = 'spir_layer.bc'
```

Comment:
Better name would be spir_wrapper.ll, also the script would be more descriptive as generate_spir_wrapper.py.

Reply:
OK, will rename